index.html

```diff
@@ -4683,82 +4683,134 @@ <h2>Internationalization Considerations</h2>
     </p>
 
     <section class="informative">
-      <h3>Language Declarations</h3>
+      <h3>Language and Direction</h3>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-for presentation, accessibility, and further processing to specify the language
-of the text, as follows:
+Data publishers are strongly encouraged to use the design pattern described
+in this section when expressing simple internationalized strings in
+JSON, JSON-LD, or similar data formats to ensure the broadest interoperability
+across syntaxes. The design pattern is a result of extensive collaboration
+between the W3C Internationalization Activity, the Linked Data Community, and
+developers that primarily utilize JSON and CBOR to publish information.
       </p>
 
-      <ul>
-        <li>
-[[!JSON-LD]] supports the expression of language information using the
-<code>@language</code> feature, as described in [[!JSON-LD]] 1.1,
-<a href="https://www.w3.org/TR/json-ld11/#string-internationalization">Section 4.2.3: String Internationalization</a>.
-        </li>
-        <li>
-[[!JSON-LD]] also supports the expression of language information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
+      <p>
+The general design pattern is to use the following markup template when
+expressing a text string that is tagged with a language and, optionally, a
+specific text direction:
+      </p>
+
+      <pre class="example nohighlight" title="Design pattern for internationalized text strings">
+"<a>property</a>": {
+  "value": "<span class="highlight">The string value</span>",
+  "language: "<code>LANGUAGE</code>-x-d-<code>DIRECTION</code>"
+}
+      </pre>
+
+      <p>
+An example utilizing the design pattern above is shown below to express the
+human-readable text "revoked" in the English language without specifying a
+text direction:
+      </p>
+
+      <pre class="example nohighlight" title="Internationalized English text">
+"statusReason": {
+  "value": "<span class="highlight">Revoked</span>",
+  "language: "<code>en</code>"
```

Comment:
Should the JSON-LD context mechanism and `@language` be used here instead? If we can just add fields, why not just add a direction metadata field (the `Localizable` dictionary in String-Meta, for example)??